### ISSUE TITLE
Documentation bug in MakeMaker.pm

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#1829]
Migrated from [rt.perl.org#3092](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=3092) (status was 'resolved')

Searchable as RT3092$

**Comment by p5pRT** [comment]
### From @earcanal

#### Created by @earcanal



pod refers to key 'HTMLSCRIPTPODS' but the real key is 'HTMLSCRIPTPOD'\.

<details><summary>Perl Info</summary>

```


This perlbug was built using Perl 5.00503 - $Date: 1999/05/05 19:42:40 $
It is being executed now by  Perl 5.006 - Tue Mar 28 14:30:22 GMT 2000.

Site configuration information for perl 5.006:

Configured by paul at Tue Mar 28 14:30:22 GMT 2000.

Summary of my perl5 (revision 5.0 version 6 subversion 0) configuration:
  Platform:
    osname=freebsd, osvers=3.3-release, archname=i386-freebsd
    uname='freebsd junodev 3.3-release freebsd 3.3-release #0: thu sep 16 23:40:35 gmt 1999 jkh@highwing.cdrom.com:usrsrcsyscompilegeneric i386 '
    config_args=''
    hint=recommended, useposix=true, d_sigaction=define
    usethreads=undef use5005threads=undef useithreads=undef usemultiplicity=undef
    useperlio=undef d_sfio=undef uselargefiles=define 
    use64bitint=undef use64bitall=undef uselongdouble=undef usesocks=undef
  Compiler:
    cc='gcc', optimize='-O', gccversion=2.7.2.3
    cppflags='-I/usr/local/include'
    ccflags ='-I/usr/local/include'
    stdchar='char', d_stdstdio=undef, usevfork=true
    intsize=4, longsize=4, ptrsize=4, doublesize=8
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=12
    ivtype='long', ivsize=4, nvtype='double', nvsize=8, Off_t='off_t', lseeksize=8
    alignbytes=4, usemymalloc=n, prototype=define
  Linker and Libraries:
    ld='gcc', ldflags ='-Wl,-E  -L/usr/local/lib'
    libpth=/usr/lib /usr/local/lib
    libs=-lm -lc -lcrypt
    libc=/usr/lib/libc.so, so=so, useshrplib=false, libperl=libperl.a
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=so, d_dlsymun=undef, ccdlflags=' '
    cccdlflags='-DPIC -fpic', lddlflags='-shared  -L/usr/local/lib'

Locally applied patches:
    


@INC for perl 5.006:
    /usr/local/lib/perl5/5.6.0/i386-freebsd
    /usr/local/lib/perl5/5.6.0
    /usr/local/lib/perl5/site_perl/5.6.0/i386-freebsd
    /usr/local/lib/perl5/site_perl/5.6.0
    /usr/local/lib/perl5/site_perl
    .


Environment for perl 5.006:
    HOME=/home/paul
    LANG (unset)
    LANGUAGE (unset)
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)
    PATH=/home/paul/bin:/bin:/usr/bin:/usr/local/bin:/usr/X11R6/bin:/usr/local/bin:/usr/local/samba/bin
    PERL_BADLANG (unset)
    SHELL=/usr/local/bin/bash


```
</details>

**Comment by p5pRT** [comment]
### From The RT System itself

closed&#8203;: @&#8203;7385 only refers to HTMLSCRIPTPODS
